HW1/P6/P6.py

```diff
@@ -0,0 +1,60 @@
+import findspark
+findspark.find()
+findspark.init('/usr/local/opt/apache-spark/libexec')
+import pyspark
+import copy
+
+sc = pyspark.SparkContext()
+
+def getWordCount(wlist):
+    result = {}
+    for i in wlist:
+        try:
+            result[i] = result[i] + 1
+        except: 
+            result[i] = 1
+    result = sorted(result.items(), key=lambda x: x[1], reverse = True)
+    return result
+
+def generateLine(word_count):
+    init_key = resultRDD.keys().takeSample(True, 1)[0] 
+    phrase = init_key[0] + ' ' + init_key[1]
+    for i in xrange(word_count - 2):
+        try:
+            new_word = resultRDD.map(lambda x: x).lookup(init_key)[0][0][0]
+        except: 
+            init_key = resultRDD.keys().takeSample(True, 1)[0]
+            new_word = resultRDD.map(lambda x: x).lookup(init_key)[0][0][0]
```

Comment:
This is good to prevent error if a bigram does not exist. PS: It is interesting because you were not randomly generating bigram out of nowhere. You were generating from the original text. Only the last bigram was possible to raise this error.